File: basement/04_for/answer/1.c
// 1つの数字を入力して、その回数だけ `Hello world !` と出力しよう

#include <stdio.h>
int main(void){
    int n, i;
    scanf("%d", &n);
    
    for (i=0; i<n; i++) {
        printf("Hello world !\n");
    }
}

File: applied/16_string2/answer/_1.java
// 入力した文字列が、"Hello" を含んでいた場合に、"World" を出力してください
// 含んでいない場合は、"Please say "Hello" !" と出力してください

import java.util.*;

public class _1 {
    public static void main(String[] args) throws Exception {
        Scanner scan = new Scanner(System.in);
        String input = scan.nextLine();
        scan.close();

        if (input.indexOf("Hello") == -1) {
            System.out.printf("Please say \"Hello\" \n");
        } else {
            System.out.printf("World\n");
        }
    }
}
